modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/application/ApplicationDescriptorFactoryTestCase.java

```diff
@@ -91,6 +91,11 @@ private static File getResourceFile(String resource) {
                                                                         .getAbsolutePath());
 
   @Rule
+  public SystemProperty remoteMavenRepository =
+      new SystemProperty("muleRuntimeConfig.maven.repositories.mulesoftPublic.url",
```

Comment:
I have some kind of deja-vu with this change... Why a unit test has to download artifacts from a external repo?